plutus-core/plutus-core/src/PlutusCore.hs

```diff
@@ -222,15 +209,15 @@ typecheckPipeline
     -> m (Normalized (Type TyName DefaultUni ()))
 typecheckPipeline = inferTypeOfProgram
 
-format :: (Monad m,
+format :: (AsParserErrorBundle [Char], Monad m,
```

Comment:
I wouldn't do this: this is using the prism machinery to convert to a string, whereas I think it's cleaner to just use it to convert to particular errors. If you want to format it here, just call `pretty` or whatever.

Reply:
Switched back to how this function was.